getYahooFinanceData.py

```diff
@@ -19,6 +19,9 @@
 # ^GSPC 19500103 # S&P 500
 # ^N225 19840104 # Nikkei 225
 
+# To make print working for Python2/3
+from __future__ import print_function
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [__future__ import is not the first non docstring statement](https://app.codacy.com/manual/mathieugouin/tradesim/pullRequest?prid=5336609)